src/pages/uxp/reference-js/Modules/shell/Shell.md

```diff
@@ -1,22 +1,59 @@
----
-jsDoc: true
----
 
 <a name="shell" id="shell"></a>
 
 # require('shell')
-To get an instance: `require("uxp").shell`
+To get an instance: `require("uxp").shell`<br/>
+These APIs require UXP Manifest v5 configurations. see [Launch Process](https://developer.adobe.com/photoshop/uxp/2022/guides/uxp_guide/uxp-misc/manifest-v5/#launch-process)
 
 
 
-<a name="shell-openexternal" id="shell-openexternal"></a>
+<a name="shell-openpath" id="shell-openpath"></a>
+
+## openPath(path, developerText)
+Opens the given file or folder path in the system default application.<br/>
+NOTE: UWP can access only files in the UWP App sandbox. see [File access permissions in UWP](https://docs.microsoft.com/en-us/windows/uwp/files/file-access-permissions)
```

Comment:
Information not suitable for 3P as it would require deep knowledge of the architecture of various CC apps (which might be present with experienced 3P devs, but not with beginners)

Reply:
🔴 🟡

Reply:
Tracked in #80